tests/test_comment.py

```diff
@@ -0,0 +1,19 @@
+"""This module test the counter.py module regarding comments"""
```

Comment:
All test cases should be put into the `test_counter.py` file. Once you have transferred over your test cases, please delete the `test_comment.py` file.

You should also not be testing the `hello` directory, but rather the `test_files` directory that was merged to your branch from `develop`. Please correct `test_match_comment_returns_correct_number_comments()` function to reflect this. With this change, your assert should be 3 and not 2.